### PR TITLE
web: show resource uibuttons on table view

### PR DESCRIPTION
**Pull request by landism**
For any uibuttons that specify a resource as their location, render them in the table view.
1. Add a "widgets" column iff there are any buttons.
2. Button text currently only shows up in the tooltip, to save room.

image

FWIW, I only invested in the "ton of buttons" rendering case enough to make it usable, but not good. Here's what it currently looks like:
image

### CODE REVIEW

File: web/src/OverviewTable.tsx
Comment (lizzthabet) [inline web/src/OverviewTable.tsx:371]: thanks for this comment 👌🏻

File: web/src/OverviewTable.tsx
Comment (milas) [inline web/src/OverviewTable.tsx:578]: fwiw this seems reasonable to me

**Review by milas — APPROVED** [review]

File: web/src/OverviewTable.tsx
Comment (lizzthabet) [inline web/src/OverviewTable.tsx:579]: this makes sense to me, but i'll let @SurbhiSGupta and others chime in too!

File: web/src/OverviewTable.tsx
Comment (lizzthabet) [inline web/src/OverviewTable.tsx:490]: (nitpick) it'd be nice to have this as a helper function related to the custom ui buttons code, maybe like `isResourceButton`?

**Review by lizzthabet — COMMENTED** [review]
this looks good! i had one outstanding ux question and may have some a11y suggestions based on your answer.

File: web/src/OverviewTable.tsx
Comment (landism) [reply web/src/OverviewTable.tsx:578]: yeah, it's at least fine. my main hesitations are:
1) it's inconsistent with other columns (e.g., we don't hide endpoints or podids when all of their cells are empty)
2) showing an empty column allows space for advertising the feature, though I'm not sure we'd do that at this point anyway

File: web/src/OverviewTable.tsx
Comment (landism) [reply web/src/OverviewTable.tsx:361]: Nope, not intentional. Thanks!

File: web/src/OverviewTable.tsx
Comment (landism) [reply web/src/OverviewTable.tsx:371]: Yes, for sure. Added a TODO.

File: web/src/OverviewTable.tsx
Comment (landism) [reply web/src/OverviewTable.tsx:490]: yeah! though I went with the less general `buttonsForResource` (which also does the filter) since for now that's the only way we'd use it and I'm not sure if that will change.

File: web/src/OverviewTable.tsx
Comment (landism) [reply web/src/OverviewTable.tsx:579]: I'm happy to change this later, but don't think it matters enough either way to block merging on

**Pull request merged**
